### DISCUSSION TITLE
The React Template is not bringing the local host after npm start

### DISCUSSION THREAD

**Discussion by JuanDavila1101** [nss-evening-cohort-14/evening-client-side#219] [Help Me!] [answered]
The React Template Application is not bringing the local host after doing the npm i and npm start

I have research other tickets, and I don't see this issue

research the issue with
   https://github.com/olefredrik/FoundationPress/issues/1328
   https://github.com/dvajs/dva-example-user-dashboard/issues/32
   https://discuss.dev.twitch.tv/t/note-that-the-development-build-is-not-optimized/19001

after running the npm start I get this on the terminal
Compiled successfully!

You can now view react-template in the browser.

  Local:            http://localhost:3000
  On Your Network:  http://192.168.1.69:3000

Note that the development build is not optimized.
To create a production build, use yarn build.

did a git clone git_New_Template_Repo
npm i
npm start
it compiles successfully, but it doesn't load the local browser
Tried deleting the node_modules 
reinstalling the npm i and npm install and then running npm start and get the same issue
npm start 
it compiles successfully, but it doesn't load the local browser

node version is: v13.14.0
npm version is: 6.14.4

**Comment by drteresavasquez** [comment, marked as answer]
This is a machine issue. Getting more ram and clearing up space to see if that helps.
